octodns/provider/route53.py

```diff
@@ -550,14 +550,21 @@ def _healthcheck_measure_latency(self, record):
             .get('healthcheck', {}) \
             .get('measure_latency', True)
 
+    def _healthcheck_request_interval(self, record):
+        interval = record._octodns.get('route53', {}) \
+            .get('healthcheck', {}) \
+            .get('request_interval')
+        return interval if (interval in [10, 30]) else 10
```

Comment:
If these are the only two values allowed it'd be good to throw an exception or at least warning if we get something else.